static/src/javascripts/projects/common/utils/formInlineLabels.js

```diff
@@ -0,0 +1,52 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'fastdom'
+], function (
+    bean,
+    qwery,
+    $,
+    fastdom
+) {
+    /**
+     * @param {String} type 'add' or 'remove'
+     * @param {Object} $el $ element
+     * @param {String} cssClass The class to add or remove
+     * @param {Function} testFunc A callback function that should return boolean describing whether we
+     * should perform the add or remove class
+     */
+    function updateClass(type, $el, cssClass, testFunc) {
+        return function () {
+            // If we pass a boolean for test, then check if we should update the class
+            var updateClass = (testFunc !== undefined) ? testFunc() : true;
+
+            if (updateClass) {
+                fastdom.write(function () {
+                    $el[(type === 'add') ? 'addClass' : 'removeClass'](cssClass);
+                });
+            }
+        };
+    }
+
+    return {
+        init: function (el) {
+            var $el = $(el),
+                $input = $('.js-email-sub__text-input', el),
```

Comment:
Technically it should work with anything, I shouldn't have the email specific class here - will make more generic, thanks!